scripts/exglobal_cleanup.sh

```diff
@@ -113,3 +105,13 @@ if (( GDATE < RDATE )); then
 fi
 deletion_target="${ROTDIR}/${RUN}.${RDATE:0:8}"
 if [[ -d ${deletion_target} ]]; then rm -rf "${deletion_target}"; fi
+
+# Finally, delete DATAROOT.
```

Comment:
```suggestion
# sync and wait to avoid filesystem synchronization issues
sync && sleep 1

# Finally, delete DATAROOT.
```

Reply:
Reading up on `sync`, I think its use here is fine and safe.